WcaOnRails/app/models/user.rb

```diff
@@ -568,7 +568,8 @@ def notify_of_id_claim_possibility(competition)
 
   def self.find_first_by_auth_conditions(warden_conditions)
     conditions = warden_conditions.dup
-    if login = conditions.delete(:login)
+    login = conditions.delete(:login)
```

Comment:
If this is how your team codes and you're used to seeing it, it streamlines some fairly common constructs.

But I'm not used to seeing it.